introduction.md

```diff
@@ -0,0 +1 @@
+Hell my name is David and I want to join your DoJo
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.